forms/swipe.json

```
@@ -125,5 +95,10 @@
     "name": "sportDomicile",
     "body": "J’ai fait du sport à la maison",
     "icon": "🏋"
+  },
+{
+    "name": "travauxChamp",
+    "body": "J’ai travaillé aux champs" ,
```

Aucune espace n'est autorisé entre la chaine de caractères et la virgule.

```suggestion
    "body": "J’ai travaillé aux champs",
```